nntrainer/tensor/tensor_wrap_specs.h

```diff
@@ -70,6 +74,40 @@ typedef std::tuple<TensorDim, Tensor::Initializer, bool, const std::string,
                    TensorLifespan>
   VarGradSpec;
 
+/**
+ * @brief Tensor Specification which describes how this tensor should be
+ * allocated and managed
+ *
+ */
+struct TensorSpecV2 {
+  enum class RequestType {
```

Comment:
So.. every request*() make this struct and used in manager to set the memory pool and planner?

Reply:
yes, init context will have this specification to be passed to manager.